cmake/inputs/patches/ep_azuresdk/azure-storage-lite-base64.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/base64.cpp b/src/base64.cpp
+index e6dead79..55fab443 100644
+--- a/src/base64.cpp
++++ b/src/base64.cpp
+@@ -1,5 +1,8 @@
+ #include <array>
+ #include <cstring>
++//with thanks to https://github.com/Azure/azure-storage-fuse/issues/421#issuecomment-655271535
```

Comment:
Bugs are frustrating, but let's keep the tone a little more neutral in our comments :-)